multicluster/cmd/multicluster-controller/options.go

```diff
@@ -27,6 +29,8 @@ type Options struct {
 	configFile     string
 	SelfSignedCert bool
 	options        ctrl.Options
+	leader         bool
```

Comment:
i think we should be able to directly access the leader value from `Options.options` instead of storing it in this struct

Reply:
Options.options is of type ctrl.Options, that wont have leader/member flags. It only as what controller-runtime library needs to run its logic right?